packages/next/src/lib/metadata/types/metadata-interface.ts

```diff
@@ -160,15 +164,15 @@ interface Metadata extends DeprecatedMetadataFields {
   /**
    * The viewport setting for the document.
    * @example
+   * @deprecated
+   *
    * ```tsx
-   * "width=device-width, initial-scale=1"
-   * <meta name="viewport" content="width=device-width, initial-scale=1" />
    *
    * { width: "device-width", initialScale: 1 }
    * <meta name="viewport" content="width=device-width, initial-scale=1" />
    * ```
    */
-  viewport?: null | string | Viewport
+  viewport?: null | ViewportLayout
```

Comment:
for the deprecated field probably should leave the string type in place right? The field will always be null with this PR so it's not all that meaningful but no point in breaking typescript for those that have to migrate code